### PR TITLE
[MIG] remote_report_to_printer: Migration to 13.0

### PR DESCRIPTION
**Pull request by olgamarcocb**
Standard Migration

### CODE REVIEW

File: remote_report_to_printer/models/printing_action.py [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class PrintingAction(models.Model):
     _inherit = "printing.action"
 
-    @api.model
```
Comment (etobella) [inline remote_report_to_printer/models/printing_action.py]: You should need to keep this

**Review by etobella — COMMENTED** [review]
Just a small comment

File: remote_report_to_printer/models/printing_action.py (hunk 1 above)
Comment (olgamarcocb) [reply remote_report_to_printer/models/printing_action.py]: True! :wink:

**Review by etobella — APPROVED** [review]

**Review by albariera — APPROVED** [review]

**Comment by albariera** [comment]
LGTM :+1:

**Pull request merged**